tests/unit_test.py

```diff
@@ -131,19 +131,24 @@ def test_token_auth_creation():
         assert token_auth.skip_tls == True
         assert token_auth.ca_cert_path == None
 
+        os.environ["CF_SDK_CA_CERT_PATH"] = f"/etc/pki/tls/custom-certs/ca-bundle.crt"
         token_auth = TokenAuthentication(token="token", server="server", skip_tls=False)
         assert token_auth.token == "token"
         assert token_auth.server == "server"
         assert token_auth.skip_tls == False
-        assert token_auth.ca_cert_path == None
+        assert token_auth.ca_cert_path == "/etc/pki/tls/custom-certs/ca-bundle.crt"
+        os.environ.pop("CF_SDK_CA_CERT_PATH")
```

Comment:
I think this line is not necessary, further proving that priority is given to the `ca_cert_path` parameter even if the environment variable is set.

Reply:
It was failing some other unit tests since they don't expect the env var to be set